tests/Jellyfin.Naming.Tests/TV/AbsoluteEpisodeNumberTests.cs

```diff
@@ -21,7 +21,8 @@ public void GetEpisodeNumberFromFileTest(string path, int episodeNumber)
             var result = new EpisodeResolver(options)
                 .Resolve(path, false, null, null, true);
 
-            Assert.Equal(episodeNumber, result.EpisodeNumber);
+            Assert.NotNull(result);
+            Assert.Equal(episodeNumber, result!.EpisodeNumber);
```

Comment:
A better solution would be marking `episodeNumber` as nullable

Reply:
Sorry, I'm not sure I follow how that would fix it.  The warning this fixes was 

> TV/AbsoluteEpisodeNumberTests.cs(25,41): warning CS8602: Dereference of a possibly null reference.

related to `result` being dereferenced.  I could see how it could affect it if the issue was that `EpisodeNumber` was potentially null, but if `result` itself is, I would expect the problem the compiler is detecting is that when it evaluated the parameters for the `Assert.Equal` call (i.e. before we even got to the comparison to `episodeNumber`), it will throw a NullReferenceException.  Unless we can signal to the compiler that result itself won't be null, won't that issue persist?

I tried a couple changes to `episodeNumber`'s nullability, but none seemed to remove the warning.  However, I'm new to C# (though old to programming), so perhaps I'm just misunderstanding your suggestion.

Reply:
My bad, I misread the changelog